sdk/instrumentation/net/http/handler.go

```diff
@@ -45,16 +57,39 @@ func (h *handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		span.SetAttribute(key, value)
 	}
 
-	span.SetAttribute("http.url", r.URL.String())
+	url := r.URL.String()
+	span.SetAttribute("http.url", url)
+
+	// run url filters
+	for _, urlEvaluator := range h.urlFilters {
```

Comment:
I think all the filters should go up as in this line https://github.com/hypertrace/goagent/pull/83/files#diff-c9de109efb7aa8920966082d6c5481c3324292b143637ff5c5f3f2174a5e1b9dR52 we are passing the request to the userland handler.

Reply:
The link doesn't take me anywhere for some reason. Can you elaborate what you mean by going up?